mmv1/products/compute/terraform.yaml

```diff
@@ -901,10 +901,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     examples:
       - !ruby/object:Provider::Terraform::Examples
         name: "external_http_lb_mig_backend_custom_header"
-        primary_resource_id: "google_compute_global_forwarding_rule"
+        primary_resource_id: "default"
         vars:
           xlb_network_name: "l7-xlb-network"
           backend_subnet_name: "l7-xlb-subnet"
+          address_name: "l7-xlb-static-ip"
```

Comment:
We'll need to set `ignore_read_extra` on this example (https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/provider/terraform/examples.rb#L117-L119) against the `ip_address` field, as the API returns the address value (ip) when specified with the address id.

Reply:
Hi @rileykarson  - Thank you for your comments! As suggested , will add `ip_address` into ignore read extra & check tests.